src/components/Message.vue

```diff
@@ -206,22 +254,93 @@ export default {
 				},
 			})
 		},
+		onToggleSeen() {
+			this.$store.dispatch('toggleEnvelopeSeen', this.envelope)
+		},
+		onDelete(e) {
+			// Don't try to navigate to the deleted message
+			e.preventDefault()
```

Comment:
copy-paste error? I don't get why this is needed here. the action is just a button. it won't navigate by default

Reply:
also this code below is unrelated, no?

Reply:
you're right the `e.preventDefault` isn't necessary here.

Though, I think the code below is needed: If the user deletes the currently displayed message we need to show the next/previous one.

Reply:
Yep, but it's basically duplicated code …

merging now, but this has to be cleaned up later on